project_generator/templates/makefile_gcc.tmpl

```diff
@@ -40,7 +40,7 @@ TARGET_EXT =
 TARGET_EXT = a
 {% endif %}
 
-LD_SCRIPT = {{linker_file}}
+LD_SCRIPT = {% for file in linker_file %} {{file}} {% endfor %}
```

Comment:
Except that we are using -T, which says (in laymen terms) replace your current script with the one specified.